src/components/app/App.js

```diff
@@ -3,16 +3,19 @@ import Main from '../main/Main'
 import Home from '../home/Home'
 import Footer from '../footer/Footer'
 import FlipCard from '../card/FlipCard'
+import CreateActivityForm from '../createActivityForm/CreateActivityForm'
 import styled from 'styled-components/macro'
+import { useState } from 'react'
 
-export default function App({ activities }) {
+export default function App({ data }) {
+  const [initialActivities, setActivities] = useState(data)
```

Comment:
```suggestion
export default function App({ initialActivities }) {
  const [activities, setActivities] = useState(initialActivities)
```
after the first state update they are no longer the initial activities, so activities works well as a name for the state. Naming the imported data initialState makes the intent clearer
;)